
  0%|                                                                                                                                       | 0/20 [00:07<?, ?it/s]
Traceback (most recent call last):
  File "C:\Users\hiren\OneDrive\Desktop\work with smita\PointCloudNet\main.py", line 95, in <module>
    train(model, mlp)
  File "C:\Users\hiren\OneDrive\Desktop\work with smita\PointCloudNet\main.py", line 66, in train
    wandb.log({f"{name}.grad": wandb.Histogram(param.grad.cpu().data.numpy())}, step=epoch)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 450, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 401, in wrapper_fn
    return func(self, *args, **kwargs)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 1924, in log
    self._log(data=data, step=step, commit=commit)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 1645, in _log
    self._partial_history_callback(data, step, commit)
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\wandb_run.py", line 1517, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "C:\Users\hiren\anaconda3\envs\pcenv\lib\site-packages\wandb\sdk\interface\interface.py", line 641, in publish_partial_history
    partial_history.step.num = step
TypeError: 'tuple' object cannot be interpreted as an integer